src/pages/signin/SignInPageLayout/index.js

```diff
@@ -10,9 +10,9 @@ const propTypes = {
 const SignInPageLayout = props => (
     !props.isSmallScreenWidth
         // eslint-disable-next-line react/jsx-props-no-spreading
-        ? <SignInPageLayoutWide {...props} />
+        ? <SignInPageLayoutWide {...props}>{props.children}</SignInPageLayoutWide>
         // eslint-disable-next-line react/jsx-props-no-spreading
-        : <SignInPageLayoutNarrow {...props} />
+        : <SignInPageLayoutNarrow {...props}>{props.children}</SignInPageLayoutNarrow>
```

Comment:
These changes don't seem necessary as passing props should pass `props.children`.

Reply:
It was throwing warnings, so I added them; if you think it's okay to keep the warnings then I think it's cleaner not to have it.